components/IndoorDirections/Dijkstra/DijkstraAlgorithm.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Implementation of Dijkstra's shortest path algorithm
```

Comment:
Be careful, I saw that these are _constants_  and not functions with the keyword _function_ so the tests might not behave the way you want them too.

Reply:
This is a stateless function. The test class for Dijksta works well

Reply:
GREAT!